test/SILOptimizer/cse.sil

```diff
@@ -1244,6 +1244,27 @@ bb3:
   return %20 : $()
 }
 
+// CHECK-LABEL: sil @cse_open_existential2 :
+// CHECK: open_existential_ref
+// CHECK-NOT: open_existential_ref
+// CHECK-LABEL: } // end sil function 'cse_open_existential2'
+sil @cse_open_existential2 : $@convention(thin) (@guaranteed Proto, Bool) -> () {
```

Comment:
Can you add a better test name for this. It would be great to in the test say more of what you are talking about in the commit msg so your narrative from the commit msg is more in the commit itself.